testing/helpers/chroot-debian.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/bash
 
+: "${RELEASE:=bullseye}"
```

Comment:
Can the `RELEASE` and possibly the other environment variables be added to the help output of `setup.sh`? We won't be able to enumerate through the various values / show default values, but at least the variable names + description of what they do can be documented.

Reply:
Done.